C-Plus-Plus/other/CountSetBits.cpp

```diff
@@ -0,0 +1,28 @@
+#include <bits/stdc++.h>
+using namespace std;
+ 
+unsigned int countSetBits(unsigned int n)
+{
+    unsigned int count = 0;
+    while (n) {
+        count += n & 1;
+        n >>= 1;
+    }
+    return count;
+}
+ 
+int main()
+{
+    int n;
+    cout << "Enter a number" << endl;
+    cin >> n;
+    cout <<"The number of set bits or 1s is: " countSetBits(n);
+    return 0;
+}
```

Comment:
Code plagiarized from [here](https://www.geeksforgeeks.org/count-set-bits-in-an-integer/)